### PR TITLE
Start the README

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Probably could be expanded still.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+These can be run using `docker-compose up`.
+
+- `api-dummy`: A dummy implementation of the API, containing sample XML content in multiple languages from eLife, Hindawi, SciELO, International Journal of Microsimulation and Wikipedia, and corresponding RELAX NG schemas for each source. Can be viewed at http://localhost:8081/articles.
+- `journal`: A basic implementation of the public-facing site, that lists and displays articles from an API implementation in multiple languages. Can be viewed at http://localhost:8080/.
+
+## Experiments
+
+- `events`: Experiments into PHP and Python implementations of a RabbitMQ-based event bus.
+- `workflow`: An experiment into using Airflow as a workflow.
```
Comment (seanwiseman) [inline README.md]: Currently two choices, the second (custom implementation) does not use `Airflow` at all.

Comment (thewilkybarkid) [reply README.md]: #11? Trying to only show things that are merged.

Comment (seanwiseman) [reply README.md]: Understood

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+It contains:
+
+## Applications
+
+These can be run using `docker-compose up`.
+
+- `api-dummy`: A dummy implementation of the API, containing sample XML content in multiple languages from eLife, Hindawi, SciELO, International Journal of Microsimulation and Wikipedia, and corresponding RELAX NG schemas for each source. Can be viewed at http://localhost:8081/articles.
+  - XML content is stored in `data`
+  - Schemas are stored in `public/schemas`
+- `journal`: A basic implementation of the public-facing site, that lists and displays articles from an API implementation in multiple languages. Can be viewed at http://localhost:8080/.
```
Comment (giorgiosironi) [inline README.md:14]: don't remember how the two talk to each other, if they do; worth explaining

**Pull request merged**